packages/acisfp_check/post_check_logs.py

```diff
@@ -10,4 +10,5 @@
                     'AstropyDeprecationWarning: out/states.dat already exists.',
                     'AstropyDeprecationWarning: headout/states.dat already exists.',
                     'ErfaWarning: ERFA function "dtf2d" yielded 2 of "dubious year',
-                    'ErfaWarning: ERFA function "utctai" yielded 2 of "dubious year'])
+                    'ErfaWarning: ERFA function "utctai" yielded 2 of "dubious year',
+                    'RuntimeWarning: invalid value encountered in cast'])
```

Comment:
I don't entirely understand why this didn't get filtered from pytest.ini -- I gave up and stuck it here.

Reply:
I specifically did not want to ignore this warning because it can happen generally and I thought it could be an issue.

Most of the ignores we have added are warnings in code that is not our reponsibility, and will most probably not show in our code.

Reply:
Right, but similarly, I don't want to see the acis warnings every day -- I almost missed that it looks like there's a new sparkles issue because of the noise.  So it seems like we can either add these to the pytest.ini, add it to the ignore for the specific applications in post_check_logs.py, or remove acis_thermal_check and acisfp_check from the overnight testing.

Reply:
here is ok, because it applies only to acis.